src/main/java/ca/corefacility/bioinformatics/irida/pipeline/results/updater/impl/BioHanselSampleUpdater.java

```diff
@@ -69,18 +70,16 @@ public void update(Collection<Sample> samples, AnalysisSubmission analysis) thro
 			throw new PostProcessingException(
 					"Expected one sample; got '" + samples.size() + "' for analysis [id=" + analysis.getId() + "]");
 		}
-		final Sample sample = samples.iterator()
-				.next();
+		final Sample sample = samples.iterator().next();
 
-		AnalysisOutputFile aof = analysis.getAnalysis()
-				.getAnalysisOutputFile(BIO_HANSEL_RESULTS_FILE);
+		AnalysisOutputFile aof = analysis.getAnalysis().getAnalysisOutputFile(BIO_HANSEL_RESULTS_FILE);
 
 		Path filePath = aof.getFile();
 
 		Map<String, MetadataEntry> stringEntries = new HashMap<>();
 		try {
-			@SuppressWarnings("resource") String jsonText = new Scanner(
-					new BufferedReader(new FileReader(filePath.toFile()))).useDelimiter("\\Z")
+			@SuppressWarnings("resource")
+			String jsonText = new Scanner(new BufferedReader(new FileReader(filePath.toFile()))).useDelimiter("\\Z")
```

Comment:
This one is like what is was before the formatting in eclipse

Reply:
Updated intellij format file in ca919b6 to match eclipse formatting.